bindings/python/setup.py

```diff
@@ -15,16 +18,26 @@ def get_pymongoarrow_version():
 
 
 def get_extension_modules():
-    modules = cythonize(['pymongoarrow/*.pyx',
-                         'pymongoarrow/libbson/*.pyx'])
-    for module in modules:
+    arrow_modules = cythonize(['pymongoarrow/*.pyx'])
+    libbson_modules = cythonize(['pymongoarrow/libbson/*.pyx'])
+
+    for module in libbson_modules:
         module.libraries.append('bson-1.0')
-    return modules
+
+    for module in arrow_modules:
+        module.include_dirs.append(np.get_include())
+        module.include_dirs.append(pa.get_include())
+        module.libraries.extend(pa.get_libraries())
+        module.library_dirs.extend(pa.get_library_dirs())
+        if os.name == 'posix':
+            module.extra_compile_args.append('-std=c++11')
+
+    return arrow_modules + libbson_modules
 
 
 setup(
     name='pymongoarrow',
     version=get_pymongoarrow_version(),
     packages=find_packages(),
     ext_modules=get_extension_modules(),
-    setup_requires=['cython >= 0.29'])
+    setup_requires=['cython >= 0.29', 'pyarrow >= 3', 'numpy >= 1.16.6'])
```

Comment:
Let me see if I have this right. So we need pyarrow+numpy to build+install the wheel but the wheel can be installed and used even without pyarrow+numpy? Or are we missing a necessary `install_requires`?

Reply:
We are missing the `install_requires` - we will depend on pymongo and pyarrow (pyarrow has a dependency on numpy). Pandas would need to be installed manually if the user intends to use that.

Reply:
Can we add `install_requires=['pyarrow >= 3']`?

Reply:
Done.